Revit_Core_Engine/Compute/CheckFamilyMapUsage.cs

```diff
@@ -40,8 +40,8 @@ public static partial class Compute
         [Description("Checks whether all Revit family names included in FamilyMaps relevant to a given BHoM type are present in the Revit document. If not, a warning is raised.")]
         [Input("bHoMType", "BHoM type to be checked against unused family maps.")]
         [Input("document", "Revit document to be checked against unused family maps.")]
-        [Input("settings", "ParameterSettings containing the ParameterMaps to be checked.")]
-        public static void CheckFamilyMapUsage(this Type bHoMType, Document document, ParameterSettings settings)
+        [Input("settings", "MappingSettings containing the ParameterMaps to be checked.")]
+        public static void CheckFamilyMapUsage(this Type bHoMType, Document document, MappingSettings settings)
```

Comment:
I suspect this is causing the `null-handling` failure because this will be a 'new' method. If not all the inputs are providing adequate `null` protection, this would register a fail that doesn't exist on `master`/`main` and thus fail the check.